odoo/tools/mail.py

```diff
@@ -639,6 +639,13 @@ def email_escape_char(email_address):
 def decode_message_header(message, header, separator=' '):
     return separator.join(h for h in message.get_all(header, []) if h)
 
+def contact_formataddr(pair, charset='utf-8'):
+    """The wrapper around formataddr that handles the case where the name contains '@' symbol"""
```

Comment:
https://peps.python.org/pep-0257/

```suggestion
    """The wrapper around formataddr that handles the case where the name contains '@' symbol."""
```